docs/index.rst

```diff
@@ -1,14 +1,125 @@
-Unified Workflow
-================
-
-TBD
+.. include:: sections/links.rst
 
 .. toctree::
    :hidden:
 
    sections/user_guide/index
    sections/contributor_guide/index
 
+
+Unified Workflow
+================
+
+``uwtools`` is an open-source modern Python package that helps automate common tasks needed for many standard numerical weather prediction (NWP) workflows and wraps Unified Forecast System (UFS) components in a highly configurable run-time interface (a driver) that maximizes flexibility, interoperability, and usability for any UFS Application.
+
+If you're interested in contributing, check out the :doc:`Contributor Guide <sections/contributor_guide/index>`.
+
+For users who want to start using the toolbox and framework, take a peek at our :doc:`User Guide <sections/user_guide/index>`.
+
+.. contents:: On this page
+   :depth: 2
+   :local:
+
+
+
+The Tools
+---------
+
+All our tools are available by command line interface (CLI), with select operations supported in the Python API. The API will also give you access to the underlying object creation to do with those what you please. We hope these options will let you integrate the package into your pre-existing bash and Python scripts, and give you some handy tools to use in your day-to-day work with running NWP systems.
+
+Configuration Management
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+| **CLI**: ``uw config -h``
+| **API**: ``import uwtools.api.config``
+
+The uw config tool suite helps you compare, transform, modify, and even validate your configuration files. The package supports YAML, shell, Fortran namelist, and INI file formats. Each of these formats allows you to specify values as Jinja2 templates to enable referencing other values for direct substitution or derived information (math, path building, string manipulation, etc.).
+
+Compare Mode
+............
+
+When the Linux diff tool just doesn't work for comparing unordered namelists with mixed-case keys, this is your go-to! It also works on the other file types, but the Fortran namelists are the *real* catalyst behind this gem!
+
+| See CLI examples here: :any:`Examples<compare_configs_cli_examples>`
+| This mode is not supported in the API.
```

Comment:
I really thought we left some things out per our conversation in person/ in Slack. So things that users might want a tool for, but there would be no reason to automate that regularly.

Reply:
IIRC those conversations were about what operations we needed in-memory alternatives for -- like, which operations need to take `dict` or `Config` arguments instead of file-path arguments. And we did scale these back, opting to stick with files for some operations. Maybe that's what you're thinking of?

Reply:
You're totally right. Thanks for the memory jog. I've removed the "not supported in API" lines for sure.